src/integrationTest/java/uk/gov/hmcts/reform/hmc/config/MessageSenderToTopicConfigurationIT.java

```diff
@@ -8,17 +8,13 @@ class MessageSenderToTopicConfigurationIT extends BaseTest  {
 
     @Test
     void shouldSuccessfullyProcessRequest() {
-        MessageSenderToTopicConfiguration messageSenderToTopicConfiguration =
```

Comment:
The tests in this class don't actually verify anything, and in integration tests we never use a real service bus anyway, so nothing lost in these two tests by using the mock bean (especially for the sake of dropping the run times of the project's integration tests from an hour to less than 2 mins).. however the tests in this class should be re-reviewed properly at some point to see if we can write something that is more meaningful.